packages/cxl-ui/scss/cxl-dashboard-header.scss

```diff
@@ -1,5 +1,6 @@
 @use "~@conversionxl/cxl-lumo-styles/scss/mq";
 @use "~@conversionxl/cxl-lumo-styles/scss/mixins";
+@import "~@conversionxl/cxl-lumo-styles/scss/typography";
```

Comment:
Yesterday I read about `@use` vs `@import` and recommendation was to use `@use` because performance is better. Could you explain why `@import` seems to be better choice here?

Reply:
I wasn't familiar with `@use` and assumed that it was meant only for functions or variables. According to this: https://sass-lang.com/documentation/at-rules/use/ `@use` will only be imported once and may not work for our use case.